cloudca/resource_cloudca_load_balancer_rule.go

```diff
@@ -145,17 +145,17 @@ func readLbr(d *schema.ResourceData, meta interface{}) error {
 		return handleLbrNotFoundError(err, d)
 	}
 
-	d.Set("name", lbr.Name)
-	d.Set("public_ip_id", lbr.PublicIpId)
-	d.Set("network_id", lbr.NetworkId)
-	d.Set("instance_ids", lbr.InstanceIds)
-	d.Set("algorithm", lbr.Algorithm)
-	d.Set("protocol", lbr.Protocol)
-	d.Set("public_port", lbr.PublicPort)
-	d.Set("private_port", lbr.PrivatePort)
-	d.Set("public_ip", lbr.PublicIp)
-	d.Set("stickiness_method", lbr.StickinessMethod)
-	d.Set("stickiness_params", lbr.StickinessPolicyParameters)
+	_ = d.Set("name", lbr.Name)
```

Comment:
This is kind of ugly but I guess if the linter is asking for it then I have no problem with that.